packages/flutter_test/test/utils/leaking_classes.dart

```diff
@@ -1,54 +0,0 @@
-// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
I'm guessing this logic was upstreamed to leak_tracker?

Reply:
yes!

Reply:
nice